auth-web/src/App.vue

```diff
@@ -117,27 +107,16 @@ export default class App extends Mixins(NextPageMixin) {
     return this.$route.meta.showNavBar
   }
 
-  private setupNavigationBar (): void {
-    this.navigationBarConfig = {
-      titleItem: {
-        name: 'Business Registry',
-        url: `/home`,
-        meta: {
-          requiresAuth: false,
-          requiresAccount: false
-        }
-      },
-      menuItems: [
-        {
-          name: 'Manage Businesses',
-          url: `/account/${this.currentAccountSettings?.id || '0'}/business`,
-          meta: {
-            requiresAuth: true,
-            requiresAccount: true
-          }
-        }
-      ]
-    }
+  /** The route breadcrumbs list. */
+  get breadcrumbs (): Array<BreadcrumbIF> {
+    const breadcrumbs = [...(this.$route?.meta?.breadcrumb || [])]
+
+    // Apply dynamic return crumb depending on user role
+    // This will be replaced when we have our UBER dashboard / high level marketing page
+    const defaultCrumb = this.currentUser?.roles?.includes(Role.Staff) ? StaffDashboardBreadcrumb : HomeBreadCrumb
+    if (this.$route?.name === 'business') { breadcrumbs.unshift(defaultCrumb) }
```

Comment:
Move block content to new line (or don't use curly braces) (but I prefer the former).

Reply:
will do!

Reply:
Done in next commit